Makefile

```diff
@@ -116,16 +116,19 @@ NOARG: world
 .PHONY: NOARG help noconfig submake
 
 help:
-	@echo "Please use either"
+	@echo "Please use either:"
 	@echo "   ./configure"
 	@echo "   make world"
 	@echo "   make install"
 	@echo "   make clean"
 	@echo "or make archclean"
-	@echo
 	@echo "For make to be verbose, add VERBOSE=1"
-	@echo
 	@echo "If you want camlp{4,5} to generate human-readable files, add READABLE_ML4=1"
```

Comment:
Not related to this PR, but since you are changing the lines just below, can you replace `camlp{4,5}` with `camlp5` (support of `camlp4` being discontinued).